### PR TITLE
Error if RD2/AdvMap is not enabled with RD fields

### PR DESCRIPTION
**Pull request by KietTruongSFDO**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -1617,6 +1617,22 @@
         <shortDescription>Warning displayed on Customizable Rollups page when enhanced RDs are enabled</shortDescription>
         <value>&lt;b&gt;WARNING: You have Enhanced Recurring Donations enabled. If you disable Customizable Rollups, rollups to Recurring Donations will no longer function.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_DisabledAdvancedMappingError</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error for DIs with populated RD fields when Advanced Mapping is disabled.</shortDescription>
```
Comment (KietTruongSFDO) [inline src/labels/CustomLabels.labels:1625]: Hi @sam-knox, do these error messages look ok?

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -1617,6 +1617,22 @@
...
+        <value>You must enable Advanced Mappings before you can process Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>RD2_DisabledRD2Error</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error for DIs with populated RD fields when RD2 is disabled.</shortDescription>
+        <value>You must enable Enhanced Recurring Donations before you can process Recurring Donations</value>
```
Comment (sam-knox) [inline src/labels/CustomLabels.labels:1634]:
```suggestion
        <value>You must enable Enhanced Recurring Donations before you can process Recurring Donations.</value>
```

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -1617,6 +1617,22 @@
         <shortDescription>Warning displayed on Customizable Rollups page when enhanced RDs are enabled</shortDescription>
         <value>&lt;b&gt;WARNING: You have Enhanced Recurring Donations enabled. If you disable Customizable Rollups, rollups to Recurring Donations will no longer function.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_DisabledAdvancedMappingError</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error for DIs with populated RD fields when Advanced Mapping is disabled.</shortDescription>
+        <value>You must enable Advanced Mappings before you can process Recurring Donations</value>
```
Comment (sam-knox) [inline src/labels/CustomLabels.labels:1626]:
```suggestion
        <value>You must enable Advanced Mapping before you can process Recurring Donations.</value>
```

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (sam-knox) [reply src/labels/CustomLabels.labels:1625]: @KietTruongSFDO - Just added a couple of minor tweaks. Otherwise good to go!

**Review by callen-sfdo — APPROVED** [review]
Looks good!

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (KietTruongSFDO) [reply src/labels/CustomLabels.labels:1625]: @sam-knox  updated those changes.

**Pull request closed without merging**